simdata/simulate_networks.py

```diff
@@ -129,6 +137,13 @@ def simulate_network_reg(n, p, uncorr_frac, num_networks, noise_stdev=0,
     The noise_stdev parameter controls the Gaussian noise added to the
     real-valued labels (defaults to 0; i.e. labels are an exact linear
     combination of the features, with weights given by B).
+
+    Network noise parameters:
+
+    add_frac: percentage of non-existing edges to add
+    remove_frac: percentage of existing edges to remove
+    add_only_uncorr: boolean, if true only add edges between correlated
```

Comment:
Correlated to what? I'm not understanding how an edge can be between correlated and uncorrelated features, because my brain jumps to the nodes being correlated to each other.

Reply:
In these experiments, I'm generating the labels/target variables using only a subset of the features (so the subset of features used to generate the labels are "correlated" with the labels, and the others are "uncorrelated" i.e. randomly sampled without any correlation to the labels). Note that this is unrelated to features/nodes that are connected in the network.

Casey and I hypothesized that adding edges to the network connecting two of these correlated features in the network was causing a disproportionately large effect on the results, since each "group" of correlated features is already a fully connected network. So, I tried connecting correlated features with uncorrelated features to see if this would attenuate the effect. (It didn't really, which I'm going to continue to explore in my next set of experiments)